### PR TITLE
PORTALS-2140: rework registration step 2 page

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Screen Shot 2022-03-22 at 12 52 04 PM

With some
Screen Shot 2022-03-22 at 12 53 09 PM
 basic responsive layout:

File: src/style/_core.scss [hunk 1]
```diff
@@ -1,7 +1,38 @@
 #body {
+
     .btn {
         border-radius: 100px;
         padding-left: 20px;
         padding-right: 20px;
     }
+    .form-group.required .form-label:after {
```
Comment (jay-hodgson) [inline src/style/_core.scss:8]: General purpose css classes.  If you add "required" to a form group, then this asterisk will be present.

File: src/style/_core.scss [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
         border-radius: 100px;
         padding-left: 20px;
         padding-right: 20px;
     }
+    .form-group.required .form-label:after { 
+        content: "*";
+        margin-left: 2px;
+        top:0px;
+    }
+    .bootstrap-4-backport .form-control {
```
Comment (jay-hodgson) [inline src/style/_core.scss:13]: Overriding form control styling for entire app.

File: src/style/_core.scss [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
+        background: #E5E5E5;
+        padding-top: 100px;
+        padding-bottom: 100px;
+        .container {
+            background: #FEFEFE;
+            box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
+            padding: 80px;
+        }
+    }
+    .buttonsContainer {
```
Comment (jay-hodgson) [inline src/style/_core.scss:33]: To layout buttons on the bottom-right of a container.

File: src/style/_core.scss [hunk 4]
```diff
@@ -1,7 +1,38 @@
...
+        &:disabled {
+            background: #e9ecef;
+        }
+    }
+    .blue-background {
+        min-height: 100vh;
+        background: #E5E5E5;
+        padding-top: 100px;
+        padding-bottom: 100px;
+        .container {
```
Comment (jay-hodgson) [inline src/style/_core.scss:27]: Styling the Bootstrap container class to look like a "card"

**Review by yjyoo773 — APPROVED** [review]
Looks good! Thank you for the extra comments.

**Pull request merged**